src/duckling/game/index.ts

```diff
@@ -1,8 +1,35 @@
-import {AttributeComponentService} from '../entitysystem/attribute-component.service';
+import {
+    AttributeComponentService,
+    AttributeDefaultService,
+    EntityBoxService,
+    EntitySystemService,
+    EntityPositionSetService
+} from '../entitysystem';
+
+import {EntityDrawerService} from '../canvas/drawing/entity-drawer.service';
+import {defaultPosition, POSITION_KEY} from './position/position-attribute';
+import {setPosition} from './position/set-position';
+import {defaultCollison, COLLISION_KEY} from './collision/collision-attribute';
 import {PositionComponent} from './position/position.component';
 import {CollisionComponent} from './collision/collision.component';
+import {drawCollision} from './collision/collision-drawer';
+import {collisionBoundingBox} from './collision/collision-box';
+
```

Comment:
A comment for this would be nice to fully explain the details of what it's doing.